problems/objects.js

```diff
@@ -61,7 +71,10 @@ const appleSet = () => {};
  *
  */
 
-const adamAndEveApples = () => {};
+const adamAndEveApples = (appleCountByName) => {
+  sum = appleCountByName.Adam + appleCountByName.Eve;
```

Comment:
Can return appleCountByName.Adam + appleCountByName.Eve; instead of saving to a variable b/c we never use the variable.